mmpose/datasets/datasets/top_down/topdown_mpii_dataset.py

```diff
@@ -248,3 +260,13 @@ def evaluate(self, outputs, res_folder, metric='PCKh', **kwargs):
         name_value = OrderedDict(name_value)
 
         return name_value
+
+    def _sort_and_unique_bboxes(self, kpts, key='bbox_id'):
```

Comment:
move it into base dataset?

Reply:
For different datasets, this function might be different.

Reply:
OK. I see.